lib/zora/createSongToken.tsx

```diff
@@ -1,19 +1,40 @@
-import { CHAIN, SONG_MARKET_CONTRACT } from '../consts';
+import { mainnet } from 'viem/chains';
+import {
+  CHAIN,
+  MIN_MINTS_FOR_SONG_MARKET,
+  REFFERAL_ADDRESS,
+  SONG_MARKET_CONTRACT,
+  ZORA_MINT_FEE,
+} from '../consts';
+import getPublicClient from '../viem/getPublicClient';
 import getSongMarketWalletClient from '../viem/getSongMarketWalletClient';
 import creatorClient from './getCreatorClient';
 
+const ensName = 'sonatatips.eth';
+
 const createSongToken = async (songLink: string) => {
-  const walletClient = getSongMarketWalletClient(CHAIN);
-  const account = walletClient.account;
-  const tokenMetadataURI = `https://test-sonata.vercel.app/api/metadata?songLink=${songLink}`;
-  const zoraResponse = await creatorClient.create1155OnExistingContract({
-    contractAddress: SONG_MARKET_CONTRACT,
-    token: {
-      tokenMetadataURI,
-    },
-    account: account.address,
-  });
   try {
+    const viemClient = getPublicClient(mainnet);
+    const walletClient = getSongMarketWalletClient(CHAIN);
+    const account = walletClient.account;
+    const tokenMetadataURI = `https://test-sonata.vercel.app/api/metadata?songLink=${songLink}`;
+    const zoraResponse = await creatorClient.create1155OnExistingContract({
+      contractAddress: SONG_MARKET_CONTRACT,
+      token: {
+        createReferral: REFFERAL_ADDRESS,
+        tokenMetadataURI,
+        salesConfig: {
+          type: 'timed',
+          marketCountdown: BigInt(24 * 60 * 60),
+          erc20Name: 'sonata song market',
+          erc20Symbol: 'SONG',
+          minimumMarketEth: MIN_MINTS_FOR_SONG_MARKET * ZORA_MINT_FEE,
+        },
+        payoutRecipient: await viemClient.getEnsAddress({ name: ensName }),
```

Comment:
this is the same value with fewer promises right?
```suggestion
        payoutRecipient: REFFERAL_ADDRESS,
```